src/pkg/services/m365/groups.go

```diff
@@ -33,8 +34,9 @@ func GroupByID(
 	ctx context.Context,
 	acct account.Account,
 	id string,
+	counter *count.Bus,
```

Comment:
this is an API that SDK users may interact with. Is this going to cause issues if they update the version of corso that they're using?

Reply:
It'll change the api signature, yes.

Reply:
is there a way around that? API updates are kind of a hassle and can land on people unexpectedly when they try to update corso

Reply:
we could revoke the capacity for callers to pass in their own counter.  I suppose I'll do that.  If anyone asks for it we can add it back later.

Reply:
alrighty, that's done